.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
The entry `db.sqlite3` has an extra space at the end. It should be `db.sqlite3` without the trailing space to ensure it is correctly ignored by Git.